docs/index.rst

```diff
@@ -7,7 +7,8 @@ Solidity
     :align: center
 
 Solidity is a contract-oriented, high-level language for implementing smart contracts.
-The syntax is similar to that of JavaScript and it is designed to target the Ethereum Virtual Machine (EVM).
+It was influenced by C++, Python and JavaScript
```

Comment:
I guess the idea for the order here is that type system came from C++, inheritance from Python and some syntax from JS?

Reply:
Exactly, only `var`, `function` are actually javascript.

Reply:
Some of this could be described, but it doesn't really make any difference. It is in flux anyway.